apps/charterafrica/src/components/Tool/Tool.js

```diff
@@ -62,7 +62,10 @@ const Tool = React.forwardRef(function Tool(props, ref) {
                 <RichTypography color="neutral.dark" variant="h2SemiBold">
                   {name}
                 </RichTypography>
-                <Link href={link.href}>
+                <Button
+                  component={externalLink?.href ? Link : undefined}
+                  href={externalLink?.href}
+                >
```

Comment:
Why the switch from Link to Button? If there is no external link, we'll just not show the link/icon. What is the point of showing a button that does nothing when clicked?